src/Microsoft.IdentityModel.Logging/SecurityArtifact.cs

```diff
@@ -14,44 +14,100 @@ namespace Microsoft.IdentityModel.Logging
     /// </remarks>
     internal struct SecurityArtifact : ISafeLogSecurityArtifact
     {
+        private const string _scrubbedArtifact = "#ScrubbedArtifact#";
```

Comment:
nit; Underscore is not needed.

Reply:
This is the style of the library at large, private variables are prepended with an underscore. Looks like we do both, so it's not consistent. Given the library doesn't have a consistent style there's nothing obvious to snap to, leaving for now. Also private so easy to change later.